packages/inertia-vue/src/link.js

```diff
@@ -30,6 +30,10 @@ export default {
     only: {
       type: Array,
       default: () => [],
+    },
+    headers: {
+      type: Object,
+      default: () => {},
```

Comment:
Realize now that this is actually wrong. It needs to be:

```js
default: () => ({}),
```

Right now this results in a default of `undefined`.

I'll fix. 👍

Reply:
Ops, sorry about that 🥺

Reply:
Haha all good brother! I didn't catch it either. 😊